contracts/core/RebalancingSetToken.sol

```diff
@@ -578,6 +578,21 @@ contract RebalancingSetToken is
         return units;
     }
 
+    /*
+     * Checks to make sure token is component of Set
+     *
+     * @param  _tokenAddress     Address of token being checked
+     * @return  bool             True if token is component of Set
+     */
+    function tokenIsComponent(
```

Comment:
Where do you intend on using this check again? Can the caller retrieve the current set from the token since its only a single selector?

If you still think its relevant, rename to something more appropriate; using the component/unit schema to create rebalancing sets doesn't mean we have to conform to the nomenclature for other methods. Maybe: `isCurrentSet`?

Reply:
Its used below in the issuanceOrder require checks. 

In this case, we need a common method for checking whether a component is part of a Set.

Reply:
This is fine then after we discussed yesterday, can you update the comment/param descriptions to be more effective.

`Checks to make sure address is the current set of the RebalancingSetToken. Conforms to the ISetToken Interface.`